File: crm-backend/src/main/java/edu/yavirac/crmbackend/feature/products/Product.java
package edu.yavirac.crmbackend.feature.products;

import java.sql.Date;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

import lombok.Data;

@Data
@Table("products")
public class Product {
    
    @Id
    @Column("producto_id")
    private long productoId;
    private String name;
    private String descripcion;
    private String marca;
    private String modelo;
    private String nombre;
    private String imagen;
    private Date created;
    private Date updated;
    private Boolean enable;
}

File: crm-frontend/src/app/feature/products/product.ts
export interface Products {
  productoId: number,
  name: string
}

import { ProductRelated } from "./ProductRelated";
export interface Product
 {
   productoId:number,
   name: string,
   descripcion: string,
   marca: string,
   modelo: string,
   personId: number,
   enable: boolean,
   created:Date,
   updated:Date,
   products: ProductRelated[]

}
